tools/docker/.env

```diff
@@ -24,6 +24,7 @@ SECURE_COOKIES=false
 EXPLORER_DB_NAME=explorer_dev
 EXPLORER_PGPASSWORD=explorer
 EXPLORER_SERVER_PORT=3001
+EXPLORER_COOKIE_SECRET=f1d2d2f924e986ac86fdf7b36c94bcdf32beec15
```

Comment:
Worth putting some magic in this to indicate it's a test secret? e.g: `b4d0000000000...`